crates/node-core/src/args/network_args.rs

```diff
@@ -73,6 +76,11 @@ pub struct NetworkArgs {
     /// Maximum number of inbound requests. default: 30
     #[arg(long)]
     pub max_inbound_peers: Option<usize>,
+
+    /// Soft limit for the byte size of a [`PooledTransactions`](reth_eth_wire::PooledTransactions)
+    /// response.
+    #[arg(long = "pooled-tx-response-soft-limit", value_name = "BYTES", default_value_t = DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE)]
+    pub soft_limit_byte_size_pooled_transactions_response: usize,
 }
```

Comment:
please add to the `help` for this arg, that this is a spec'd value that should only be set for experimental purposes on a testnet.